src/ClientData/include/ClientData/CallstackData.h

```diff
@@ -75,6 +76,19 @@ class CallstackData {
     }
   }
 
+  // Do a particular action for callstacks but skipping callstacks that will be rendered later in
+  // the same pixel in the screen. It assures to do the action once for callstacks in each occupied
```

Comment:
```suggestion
  // the same pixel on the screen. It assures to do the action once for callstacks in each occupied
```
Or just drop "on the screen"?

Reply:
Same below.